from exportana.transactions.exceptions.transasction_exception import TransactionException


class TraceException(TransactionException):
    pass
